test/methods.js

```diff
@@ -102,6 +102,17 @@ describe('nemo-view @methods@', function () {
       }
     });
   });
+  it('should return true/false using @Visible@Positive@ method when element present', function (done) {
+    nemo.view.simple.outBoxVisible().then(function (visible) {
+      assert.equal(visible, false);
+    });
+    nemo.view.form.fooButtonVisible().then(function (visible) {
+      assert.equal(visible, true);
+    }).then(util.doneSuccess(done), util.doneError(done));
+  });
+  it('should throw error using @Visible@Negative@ method when element not present', function (done) {
+    nemo.view.simple.notExistVisible().then(util.doneError(done), util.doneSuccess(done));
+  });
```

Comment:
This is similar to other tests where the fact that the error callback is called is enough verification. I.e. we aren't checking the specific error to pass the test. We _could_ do that here but it would require string matching on the error message and I'd prefer not to "brittle-ize" the test in this way.